tests/conftest.py

```diff
@@ -1536,6 +1537,26 @@ def finalizer():
     return CloudManager()
 
 
+@pytest.fixture()
+def rgw_obj(request):
+    return rgw_obj_fixture(request)
```

Comment:
is there a reasoning behind creating both session level and function level fixture for rgw? both for the rgw object and the bucket factory

Reply:
Consistency with existing fixtures and futureproofing.